src/api/exemptions/controllers/get-exemption.js

```
@@ -0,0 +1,38 @@
+import Boom from '@hapi/boom'
+import { StatusCodes } from 'http-status-codes'
+import { getExemption } from '../../../models/get-exemption.js'
+import { ObjectId } from 'mongodb'
+import { createTaskList } from '../helpers/createTaskList.js'
+
+export const getExemptionController = {
+  options: {
+    validate: {
+      params: getExemption
+    }
+  },
+  handler: async (request, h) => {
+    try {
+      const { params, db } = request
+
+      const result = await db
+        .collection('exemptions')
+        .findOne({ _id: ObjectId.createFromHexString(params.id) })
+
+      if (!result) {
+        return Boom.notFound('Exemption not found')
```

Hapi handlers should typically throw Boom-errors so that Hapi sets the status code correctly.

maybe replace with:
`throw Boom.notFound('Exemption not found')`

That way Hapi will send a 404 response instead of treating the raw Boom object as a 200.

With the above change you’ll need to update the test to expect a reject/throw rather than a return:

```
await expect(
  getExemptionController.handler({ db: mockMongo, params: { id: mockId } }, mockH)
).rejects.toThrow('Exemption not found')
```